webcompat/templates/new-issue.html

```diff
@@ -2,8 +2,5 @@
 {% block title %}New Issue{% endblock %}
 {%- block body -%}
 {% include "topbar.html" %}
-<main role="main">
-  <div class="wc-Loader js-Loader"></div>
```

Comment:
https://github.com/webcompat/webcompat.com/blob/master/webcompat/static/js/lib/bugform.js#L83

Is there an issue on file to make sure this doesn't regress when the refactor lands?

I know we have https://github.com/webcompat/webcompat.com/issues/1529, but that's very general... and I notice the refactor is either commenting out loader images, or removing them.

Reply:
We chatted about this in a 1:1, @zoepage is going to move the loader one level deeper.

Reply:
Just to add some context here, the loaders have been commented out to we don't forget them and I can look for them, when working on #1529.